docs/source/jupyterlite_config.json

```diff
@@ -0,0 +1,25 @@
+{
+    "LiteBuildConfig": {
+      "federated_extensions": [
+        "https://github.com/conda-forge/releases/releases/download/noarch/jupyterlab_widgets-1.0.2-pyhd8ed1ab_0.tar.bz2/jupyterlab_widgets-1.0.2-pyhd8ed1ab_0.tar.bz2",
+        "https://github.com/conda-forge/releases/releases/download/noarch/ipycanvas-0.11.0-pyhd8ed1ab_0.tar.bz2/ipycanvas-0.11.0-pyhd8ed1ab_0.tar.bz2"
```

Comment:
You probably need to remove ipycanvas from this file and point to ipyleaflet wheel and conda package instead